src/core/CCArray.mli

```diff
@@ -13,7 +21,11 @@ type 'a printer = Format.formatter -> 'a -> unit
 
 (** {2 Arrays} *)
 
-include module type of struct include Array end
+(**/**)
+external make_float : int -> float array = "caml_make_float_vect" (* compat *)
+(**/**)
+
+include module type of Array
```

Comment:
not sure what this change is about? the `struct include end` trick is pretty convenient…

Reply:
Since ``CCArray.mli`` is autogenerated I did not actually make any changes to it. Any changes result from inconsistencies betweed ``CCArray.mli`` and ``CCArrayLabels.mli``.
What is this trick of ``struct include end`` about? Should this be done the same way in ``CCArrayLabels.mli``? In that case it needs to be changed there.

Reply:
the `struct include end` has been discussed over IRC, I think. It's useful for keeping equalities between types (i.e. `'a CCArray.t = 'a Array.t` for the typechecker).

We can change it in `CCArrayLabels.mli` indeed!

Reply:
done.